client/src/routes/screens/LiveSession/ChangingRoom.tsx

```diff
@@ -177,17 +183,16 @@ const ChangingRoom = () => {
     } else {
       await joinMeeting({
         subscribeToTracksAutomatically: false,
-        userData: {
-          inPortal: true,
-        },
       });
+      setUserData({inPortal: true});
```

Comment:
My worry with this approach is that the data is not directly on joining, but hopefully asyncrounous short there after. Aaaaand at the same time we're going to remove this completely soon (I hope)

Reply:
Yea, know, but it's going to be a very short while before it's visible users are waiting in the protal